{{cookiecutter.project_slug}}/requirements/base.txt

```diff
@@ -50,8 +50,7 @@ django-redis==4.8.0
 redis>=2.10.5
 
 {% if cookiecutter.use_celery == "y" %}
-celery==3.1.25 # pyup: <4.0
-{% endif %}
+celery==4.1.0 # pyup: <5.0
```

Comment:
Apologies, I've messed up your diff while resolving a conflict. You'll need to re-add the `{% endif %}`.

Reply:
Sure, no worres.